packages/grafana-data/src/types/fieldColor.ts

```diff
@@ -1,52 +1,14 @@
-export enum FieldColorMode {
+export enum FieldColorModeId {
   Thresholds = 'thresholds',
-  Scheme = 'scheme',
+  ContinousGrYlRd = 'continuous-GrYlRd',
+  DiscreteClassic = 'discrete-classic',
+  DiscreteVibrant = 'discrete-vibrant',
```

Comment:
Maybe, to keep things consistent across terminology we use in the docs/UI, we could avoid using _discrete_ and use _palette_ instead?

Reply:
yea, I had it as a palette at first. But if we use Continuous for gradients the opposite of that is discrete. 

So the alternative I had was `Palette / Classic` etc, and `Spectrum / Green - Yellow - Red` , I felt that was more intuitive than discrete or continuous. But it's not technically correct, spectrums can be discrete or continuous.

So I do not know what is the best naming here